src/core/base/model.ts

```diff
@@ -74,7 +74,7 @@ export class UIPStateModel extends Observable {
 
     UIPStateModel.setAttribute(elements, attribute, 'transform' in cfg ? cfg.transform : cfg.value);
     this._lastModifier = modifier;
-    this.fire();
+    Promise.resolve().then(() => this.fire());
```

Comment:
It does not stack into a single call if multiple requests happen per task.

Reply:
The purpose to make something in a microtask is to collect all the changes/requests per task and process the te result change once before the current makrotask ends